named_test.go

```diff
@@ -214,7 +218,7 @@ func TestNamedQueries(t *testing.T) {
 		}
 
 		_, err = db.NamedExec(`INSERT INTO person (first_name, last_name, email)
-			VALUES (:first_name, :last_name, :email) `, slsMap)
+			VALUES (:first_name, :last_name, :email) ;--`, slsMap)
```

Comment:
Is this extra comment necessary?

Reply:
The comment here is exercising the regex in the sense that anything after the closing paren is ignored.

I could update it to something more clear like `ON CONFLICT` perhaps?  I imagine that most people are using that behavior moreso than random junk at the end.

Reply:
Happy with the comment, just did not want to add some extra load bearing requirement here.  I think it's clear from the tests later on that these are testing some boundary conditions.